rllib/agents/callbacks.py

```diff
@@ -26,8 +26,35 @@
     from ray.rllib.evaluation import RolloutWorker
 
 
+class _CallbackMeta(type):
```

Comment:
can we just use the Tune one here? or maybe there is a reason to keep a separate copy for rllib?

Reply:
Done!